charts/orchestrator/templates/tekton-tasks.yaml

```diff
@@ -182,14 +182,16 @@ spec:
 
         if [ -d "workflow/$(params.workflowId)" ]; then
           cp -r workflow/$(params.workflowId)/src/main/resources flat/$(params.workflowId)
+          cp workflow/$(params.workflowId)/LICENSE flat/$(params.workflowId)
         else
           cp -r workflow/src/main/resources flat/$(params.workflowId)
+          cp workflow/LICENSE flat/$(params.workflowId)
         fi
-        cp -r workflow/LICENSE flat/$(params.workflowId)/LICENSE
 
         if [ "$(params.convertToFlat)" == "false" ]; then
           rm -rf workflow/src/main/resources
           mv workflow/src flat/$(params.workflowId)/
+          cp workflow/LICENSE flat/$(params.workflowId)/LICENSE
```

Comment:
this one is not needed as it was already done by either https://github.com/parodos-dev/orchestrator-helm-chart/pull/223/files#diff-49c83779a886398e5bf8debb597019dbeb7fd23f9a981f697001286db33d3bedR185 or https://github.com/parodos-dev/orchestrator-helm-chart/pull/223/files#diff-49c83779a886398e5bf8debb597019dbeb7fd23f9a981f697001286db33d3bedR188